### PR TITLE
Add Mapbox Navigation `v2.5.4`, `v2.6.2` and `v2.7.3` to releases validation.

### PR DESCRIPTION
**Pull request by MaximAlien**
### Description

PR adds Mapbox Navigation `v2.5.4`, `v2.6.2` and `v2.7.3` to releases validation script.

### CODE REVIEW

File: scripts/validate_releases.sh [hunk 1]
```diff
@@ -37,12 +37,11 @@ EXPECTED_XCODEBUILD_RESULTS=( \
     [2.5.3]=${ERROR_CODE} \
     [2.6.0]=${ERROR_CODE} \
     [2.6.1]=${ERROR_CODE} \
+    [2.6.2]=${SUCCESS_CODE} \
     [2.7.0]=${ERROR_CODE} \
     [2.7.1]=${ERROR_CODE} \
     [2.7.2]=${ERROR_CODE} \
-    [2.8.0]=${SUCCESS_CODE} \
-    [2.8.1]=${SUCCESS_CODE} \
-    [2.9.0]=${SUCCESS_CODE}
```
Comment (kried) [inline scripts/validate_releases.sh:45]: It doesn't look like it should be removed.
These are the released versions.

File: .circleci/config.yml [hunk 2]
```diff
@@ -810,90 +810,8 @@ workflows:
...
-        - generate-docs-job:
-            name: Generate / Publish Documentation
-            context: Slack Orb
-            filters:
-              tags:
-                only: /^v\d+\.\d+\.\d+(-.+)?$/
-        - ios-trigger-metrics:
-            filters:
-              branches:
-                only: main
```
Comment (kried) [inline .circleci/config.yml:896]: Are really all those jobs should be removed?

**Review by kried — CHANGES_REQUESTED** [review]

File: scripts/validate_releases.sh (hunk 1 above)
Comment (MaximAlien) [reply scripts/validate_releases.sh:45]: PR is not ready for review yet. Accidentally removed draft label.

File: .circleci/config.yml (hunk 2 above)
Comment (MaximAlien) [reply .circleci/config.yml:896]: PR is not ready for review yet. Accidentally removed draft label.

**Review by jill-cardamon — APPROVED** [review]

File: scripts/validate_releases.sh [hunk 3]
```diff
@@ -35,11 +35,14 @@ EXPECTED_XCODEBUILD_RESULTS=( \
     [2.5.1]=${ERROR_CODE} \
     [2.5.2]=${ERROR_CODE} \
     [2.5.3]=${ERROR_CODE} \
+    [2.5.4]=${SUCCESS_CODE} \
```
Comment (MaximAlien) [inline scripts/validate_releases.sh:38]: Resolving dependencies for newly added patch versions are expected to pass now.

**Review by kried — APPROVED** [review]

**Pull request merged**